packages/query/src/graphql/graphql.module.ts

```diff
@@ -127,6 +128,18 @@ export class GraphqlModule implements OnModuleInit, OnModuleDestroy {
     }
   }
 
+  private setupKeepAlive(pgClient) {
```

Comment:
```suggestion
  private setupKeepAlive(pgClient: PoolClient) {
```
This will need an update import